src/MathOptSetDistances.jl

```diff
@@ -1,6 +1,7 @@
 module MathOptSetDistances
 
 import MathOptInterface
+using BlockDiagonals
```

Comment:
if we just need one or two functions from BlockDiagonal, it might be more appropriate to:
```
using BlockDiagonal: func1, func2
```

Reply:
edited